### PR TITLE
Add cancellation URL for subscriptions.

### PR DESCRIPTION
**Pull request by remybach**

### CODE REVIEW

File: lib/metrics/services.js [hunk 1]
```diff
@@ -109,6 +109,8 @@ module.exports = {
 	'demographic-api': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/user-ref-data\/[\w\-]+\//,
 	'subscription-api-prod-subscribe': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/subscriptions\/actions\/subscribe/,
 	'subscription-api-test-subscribe': /^https:\/\/subscription-api-gw-eu-west-1-test.memb.ft.com\/subscriptions\/actions\/subscribe/,
+	'subscription-api-prod-cancel': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/subscriptions\/cancel/,
+	'subscription-api-test-cancel': /^https:\/\/subscription-api-gw-eu-west-1-test.memb.ft.com\/subscriptions\/cancel/,
```
Comment (tavvy) [inline lib/metrics/services.js]: so i think the standard, although i admit it is fairly mixed in this module, is to format:
`api-name-<endpoint>-<?test>`. 
So:
`subscription-api-cancel`
`subscription-api-cancel-test`

Comment (remybach) [inline lib/metrics/services.js]: So should I update the normal subscription one below this as well then?

Comment (remybach) [inline lib/metrics/services.js]: I've assumed that the below one needed updating too. Left the pre-existing subscription ones above this though.

File: lib/metrics/services.js [hunk 2]
```diff
@@ -109,7 +109,9 @@ module.exports = {
 	'demographic-api': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/user-ref-data\/[\w\-]+\//,
 	'subscription-api-prod-subscribe': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/subscriptions\/actions\/subscribe/,
 	'subscription-api-test-subscribe': /^https:\/\/subscription-api-gw-eu-west-1-test.memb.ft.com\/subscriptions\/actions\/subscribe/,
-	'subscription-api-prod': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/subscriptions/,
+	'subscription-api-cancel': /^https:\/\/subscription-api-gw-eu-west-1-prod.memb.ft.com\/subscriptions\/cancel/,
+	'subscription-api-test-cancel': /^https:\/\/subscription-api-gw-eu-west-1-test.memb.ft.com\/subscriptions\/cancel/,
```
Comment (tavvy) [inline lib/metrics/services.js]: `subscription-api-cancel-test`

Comment (remybach) [inline lib/metrics/services.js]: Ack... will update 👍

**Pull request merged**